vm/systemSmartContracts/esdt.go

```diff
@@ -1544,24 +1532,15 @@ func (e *esdt) isSpecialRoleValidForSemiFungible(argument string) error {
 }
 
 func (e *esdt) isSpecialRoleValidForNonFungible(argument string) error {
-	currentEpoch := e.enableEpochsHandler.GetCurrentEpoch()
 	switch argument {
 	case core.ESDTRoleNFTBurn:
 		return nil
 	case core.ESDTRoleNFTCreate:
 		return nil
-	case core.ESDTRoleTransfer:
-		if e.enableEpochsHandler.IsESDTTransferRoleFlagEnabledInEpoch(currentEpoch) {
-			return nil
-		}
-		return vm.ErrInvalidArgument
-	case core.ESDTRoleNFTUpdateAttributes:
-		if e.enableEpochsHandler.IsESDTTransferRoleFlagEnabledInEpoch(currentEpoch) {
-			return nil
-		}
-		return vm.ErrInvalidArgument
-	case core.ESDTRoleNFTAddURI:
-		if e.enableEpochsHandler.IsESDTTransferRoleFlagEnabledInEpoch(currentEpoch) {
+	case core.ESDTRoleTransfer,
```

Comment:
maybe leave the list of case values on the same line?

Reply:
done